db/seeds.rb

```diff
@@ -5,3 +5,36 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+
+puts "will create trainer..."
+trainer = User.create!({
+  email: "trainer@mewbnb.com",
+  password: "password123"
+})
+
+puts "will create gym_leader..."
```

Comment:
Why are you seeding two personas? User is a unique role when we look at the schema no?

Reply:
shouldn't we differentiate between the renters and people renting? @jonathanjenin

Reply:
The renter exists as a user through Pokemon, the renting is the user, as per the schema and the routes on the stories spreadsheet. Please let me know if I'm missing something.